var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,12 +105,12 @@ class Qt(Package):
     # OpenGL hardware acceleration
     depends_on("mesa", when='@4:+mesa')
     depends_on("libxcb", when=sys.platform != 'darwin')
+    depends_on("libx11", when=sys.platform != 'darwin')
 
     # Webkit
     depends_on("flex", when='+webkit', type='build')
     depends_on("bison", when='+webkit', type='build')
     depends_on("gperf", when='+webkit')
-    depends_on("fontconfig", when='+webkit')
```

Comment:
Not sure why it was optional before; I made it mandatory because without fontconfig, qt expects fonts to be in `$prefix/lib/fonts`, which is empty and leads to windows without text.